### PR TITLE
feat(bridge): validate cowshed setup

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes https://www.notion.so/cownation/2025-06-Bridge-And-Swap-Security-and-edge-cases-20a8da5f04ca80eba713d93a74287416#20a8da5f04ca80f2acbee8093163f07c

image

# To Test

TODO: need to provide a script to change cowshed implementation and trustedExecutor

### CODE REVIEW

**Review by elena-zh — COMMENTED** [review]
Hey @shoom3301 , 
this works as described. 
However, I'm not sure that we need to block the entire UI in case this happens. 
I  mean, at least, a user should still see the UI in the 'view' mode and have possibility to see quotes. E.g. the same state like to a not connected user. 

From that UI a user can also call appzi modal and provide their feedback. as I don't think that users will get an 'immediate' help when they write to discord.

File: apps/cowswap-frontend/src/modules/cowShed/containers/InvalidCoWShedSetup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  justify-content: center;
+  z-index: 99999;
+  text-align: center;
+`
+
+export function InvalidCoWShedSetup(): ReactNode {
+  const proxyInfo = useCurrentAccountProxyAddress()
+  const isProxySetupValid = proxyInfo?.isProxySetupValid
+
+  if (isProxySetupValid !== false) return null
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/cowShed/containers/InvalidCoWShedSetup/index.tsx:28]: @shoom3301 Shouldn't we show this when it's `null` or `undefined` too?
Or at least show an intermediate `unknown` state to indicate we can't tell?
